pkg/plan/resource/rpm.go

```diff
@@ -143,6 +143,8 @@ func (p *RPM) Apply(r plan.Runner, diff plan.Diff) (bool, error) {
 	if p.DisableExcludes != "" {
 		cmd = fmt.Sprintf("%s --disableexcludes %s", cmd, p.DisableExcludes)
 	}
+
+	cmd = fmt.Sprintf("%s --setopt=obsoletes=0", cmd)
```

Comment:
Bit simpler:
```suggestion
	cmd += " --setopt=obsoletes=0"
```

Reply:
Also I think I would comment this noting why it was added

Reply:
Should this change affect all yum installs or should we isolate it to the kubelet install?